lib/modules/platform/azure/index.ts

```diff
@@ -990,3 +994,42 @@ export async function deleteLabel(
   const azureApiGit = await azureApi.gitApi();
   await azureApiGit.deletePullRequestLabels(config.repoId, prNumber, label);
 }
+
+function cachePr(pr?: GitPullRequest): void {
+  config.prList ??= [];
+  // istanbul ignore if
+  if (!pr) {
+    return;
+  }
+
+  const azurePr = getRenovatePRFormat(pr);
```

Comment:
I find the naming to be confusing. You have `pr` and `azurePr`, which are both non-descriptive. But worse: `azurePr` appears to be a Renovate internal format, so not actually an "azure" PR?

Reply:
I agree. I was confused aswell

Reply:
the pr is supposed to be scmPr and the azurePr is supposed to be our renovate type

Reply:
ill call them scmAzurePr
and renovateAzurePr?